docs/ref-manual/approx/array-hashes.md

```diff
@@ -0,0 +1,44 @@
+Array hashes
+============
+
+### Background: Non-Collision of Hashes is Essential for Storage Integrity
+
+Solidity makes intensive use of the Keccak hash function in order to create
```

Comment:
Add Keccak to `/spelling_wordlist.txt`.

Reply:
note that local builds run spell checking when you run `make`, so you can use this to spell check